modules/notifications/notifications.php

```diff
@@ -217,19 +193,33 @@ public function get_notification_footer( $post ) {
 	 * @param string $message_headers. (optional) Message headers
 	 */
 	public function send_email( $action, $post, $subject, $message, $message_headers = '' ) {
+		// Do nothing if the post isn't set.
+		$post_id = $post->ID;
+		if ( ! $post_id ) {
+			return;
+		}
 
-		// Get list of email recipients -- set them CC
-		$recipients = $this->get_notification_recipients( $post, true );
-
-		if ( $recipients && ! is_array( $recipients ) ) {
-			$recipients = explode( ',', $recipients );
+		// Ensure the email address is set from settings.
+		if ( empty( VIP_Workflow::instance()->settings->module->options->email_address ) ) {
+			return;
 		}
 
-		// ToDo: Figure out best filters to add for the email.
+		$email_recipients[] = VIP_Workflow::instance()->settings->module->options->email_address;
 
-		if ( ! empty( $recipients ) ) {
+		/**
+		 * Filter the email recipients
+		 *
+		 * @param array $email_recipients Array of email recipients
+		 * @param string $action Action being taken
+		 * @param WP_Post $post Post object
+		 */
+		$email_recipients = apply_filters( 'vw_notification_email_recipients', $email_recipients, $action, $post );
+
+		// ToDo: Should we add a filter to allow for customizing the email subject, body and headers?
```

Comment:
I think a subject and body filter would be great here.

Reply:
I avoided it at first, so that we could be prescriptive about these filters. But I'll add it in as I'd been going back and forth on this.